### PR TITLE
docs: a few tweaks for clarity and readability

### PR DESCRIPTION
**Pull request by anthonyeleven**
A few minor readability and spelling changes.

Signed-off-by: Anthony D'Atri <anthony.datri@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
@oddlittlebird, looking at this now.

File: docs/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Yarn >= 1.22.4
 
 ## Build the doc site
 
-1. In the command line, make sure you are in the docs folder: `cd docs`.
-1. Run `make docs`. This launches a preview of the docs website at `http://localhost:3002/docs/grafana/latest/` which will refresh automatically when changes to content in the `sources` directory are made.
+1. Onn the command line, first change to the docs folder: `cd docs`.
```
Comment (achatterjee-grafana) [inline docs/README.md:12]: Fix typo, "Onn"

**Review by achatterjee-grafana — COMMENTED** [review]
Thank you for contributing. Suggested fixing a typo.

File: docs/README.md (hunk 1 above)
Comment (anthonyeleven) [reply docs/README.md]: Well gee, that was embarrassing.  Amended.

**Review by achatterjee-grafana — COMMENTED** [review]
LGTM! Thank you for contributing!

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**